.travis.yml

```diff
@@ -31,16 +31,16 @@ matrix:
       env: "DOCKER_MYSQL_TYPE=mysql DOCKER_MYSQL_VERSION=5.5"
     - node_js: *lts
       env: "DOCKER_MYSQL_TYPE=mysql DOCKER_MYSQL_VERSION=5.6"
-    - node_js: *lts
-      env: "DOCKER_MYSQL_TYPE=mariadb DOCKER_MYSQL_VERSION=5.5"
```

Comment:
I understand these are EOL, but unless they are not working on Travis CI, we want to keep them here to ensure we do not break support until at least a major version bump of this module.